lms/templates/navigation.html

```diff
@@ -68,12 +68,14 @@ <h1 class="logo">
     <ol class="user">
       <li class="primary">
         <a href="${reverse('dashboard')}" class="user-link">
-          <span class="avatar"></span>
-          <span class="sr">${_("Dashboard for:")} </span> ${user.username}
+          <img alt='${_("Dashboard for:")}' class="avatar" src="${settings.STATIC_URL}images/small-header-home-icon.png">
```

Comment:
How about using the Font Awesome `fa-home` icon here, instead of an image?
http://fontawesome.io/icon/home/